src/HomeWork.java

```diff
@@ -0,0 +1,57 @@
+
+public class HomeWork {
+    public static int calculateAbsoluteDifference(int a, int b) {
+        return a - b;
+    }
+
+    public static char getFirstCharacter(String str) {
```

Comment:
Only method in the first task is supposed to be static. This one and all the others are non-static